### PR TITLE
Update 3.crossOriginEmbedderPolicy.md

### PR DESCRIPTION
**Pull request by espensgr**
Added credentialless as per the Cross-Origin-Embedder-Policy documentation, and you can use it as it solved my usecase.

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: docs/content/1.documentation/2.headers/3.crossOriginEmbedderPolicy.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Cross-Origin-Embedder-Policy: require-corp
 The `crossOriginEmbedderPolicy` header can be configured with following values.
 
 ```ts
-crossOriginEmbedderPolicy: 'unsafe-none' | 'require-corp' | false;
+crossOriginEmbedderPolicy: 'unsafe-none' | 'require-corp' | 'credentialless' | false;
```
Comment (Baroshem) [inline docs/content/1.documentation/2.headers/3.crossOriginEmbedderPolicy.md:56]: suggestion:

Could you also please add this new value to the types in the module? :)

https://github.com/Baroshem/nuxt-security/blob/abc860d50b323c84583eaa9c1f270f97808b1cdd/src/types/headers.ts#L5

**Review by Baroshem — COMMENTED** [review]
Thanks for this PR!

One suggestion from my side :)

File: docs/content/1.documentation/2.headers/3.crossOriginEmbedderPolicy.md (hunk 1 above)
Comment (espensgr) [reply docs/content/1.documentation/2.headers/3.crossOriginEmbedderPolicy.md:56]: Nice, forgot that, done :)

**Review by Baroshem — APPROVED** [review]

**Comment by Baroshem** [comment]
Thanks for that! It will be released with 1.0.0-rc.3 :)

**Pull request merged**